lib/build.gradle

```diff
@@ -34,7 +34,7 @@ versionCompatibility {
 				'0.45.2',
 				'0.46.0',
 				'0.47.0',
-				'0.48.0',
+				'0.48.1',
```

Comment:
If 0.48.0 and 0.48.1 are binary compatible (and as far as I can see from https://github.com/pinterest/ktlint/releases), I'd recommend that you keep the adapter version set to 0.48.0. Otherwise it appears as if the same adapter should be used for versions 0.47.x and 0.48.0 inclusive (see [manual for version-compatibility-gradle-plugin](https://github.com/davidburstrom/version-compatibility-gradle-plugin#:~:text=It%20is%20recommended%20that%20the%20compat%20version%20for%20each%20source%20set%20is%20the%20earliest%20version%20the%20adapter%20supports%2C%20for%20consistency)). In the test `works0_48_1` you would still verify that disabling works. If 0.48.0 is truly not supported, then it needs to be reflected in the code that selects the adapter.

Reply:
They are compatible, I've reverted out the renaming change.

Reply:
Cool!